provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -890,6 +890,17 @@ func generateSchema() schema.PackageSpec {
 							"Valid values are `AL2`, `AL2023` and `Bottlerocket`.\n\n" +
 							"Defaults to `AL2`.",
 					},
+					"bottlerocketSettings": {
+						TypeSpec: schema.TypeSpec{
+							Type: "object",
+							AdditionalProperties: &schema.TypeSpec{
+								Ref: "pulumi.json#/Any",
+							},
+						},
+						Description: "The configuration settings for Bottlerocket OS.\n" +
+							"The settings will get merged with the base settings the provider uses to configure Bottlerocket.\n\n" +
```

Comment:
Wonder if we can link to a doc or inline something about which base settings are these?